src/dsp_tools/commands/ingest_xmlupload/apply_ingest.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import glob
+from typing import cast
+
+import pandas as pd
+from lxml import etree
+
+from dsp_tools.commands.ingest_xmlupload.user_information import IngestInformation
+from dsp_tools.models.exceptions import InputError
+
+
+def get_mapping_dict_from_file() -> dict[str, str]:
+    """
+    This functions returns the information to replace the original filepaths with the identifier from dsp-ingest.
+
+    Returns:
+        dictionary with original: identifier from dsp-ingest
+
+    Raises:
+        InputError: if no file was found
+    """
+    if filepath := glob.glob("mapping-*.csv"):
+        df = pd.read_csv(filepath[0])
```

Comment:
this requires there to be only one mapping file, otherwise it will find the wrong one, which would lead the following steps go wrong. I'm not sure this is a problem (as it is unlikely that users have multiple mapping files in the directory. But alternatively, the mapping file could be passed as a CLI argument (like we do with the project json or the upload xml files), which would be more verbose but also a bit more stable.